docs/en_US/md/sessions.md

```diff
@@ -74,6 +74,11 @@ Is used when another session triggered this run using a trigger_flow action.
         "uuid": "b7cf0d83-f1c9-411c-96fd-c511a4cfa86d",
         "name": "Collect Age"
     },
+    "history": {
+        "parent_uuid": "a5b25fb0-75fd-4898-a34f-5ff14fc19078",
+        "ancestors": 3,
+        "ancestors_since_input": 1
```

Comment:
`ancestors_since_input` is the value we actually use but the other two could be handy when debugging

Reply:
Since this is just our session blob and we are going through this, why not just have it be complete instead of this summary? IE something more like:

```
"history": [{
  "session_uuid": "arst",
  "created_on": "2020---"
},{
  "session_uuid": "arst",
  "had_input": true,
  "created_on": "2020---"
}
]
```

Just brainstorming. I guess I feel like maintaining size is not a super important consideration here because these are sessions and we may as well annotate with all the information that is useful right in the session. Other thought is that maybe having a bit more information would help us more easily adapt our rules as to when we shortcircuit but perhaps not an important consideration.

Reply:
I talked myself out of this approach because 1) it would have to be trimmed because it's valid usecase to eternally spawn new sessions that wait and receive input and 2) from a debugging POV (where performance is not an issue) that data would be available by just giving each session a parent UUID and walking those.

But I'm kinda on the fence.. it works as long as the trimming length >= the limit.. hmm..

Reply:
I guess the other approach would be do we need to "keep" history that involves input? Then we can remove that concept entirely then no? Basically if there has been input in the current sprint then history is not passed through to any new sessions created. But then maybe the concept of "history" is too vague.

Reply:
But then you lose potentially interesting information if there is input.. or ability to re-use this information with slightly different rules for loop detection.. we'd probably call this something like `loop_detection_history`

I think I've talked myself into keeping this as is and if in the future we want more debugging info.. it's a dict so could add an array field like `last_sessions` or whatever.

Also if we store this as a blob called `history` on the flow start then we can add whatever there without DB changes